src/osmf/js/osmf_flash.js

```diff
@@ -391,6 +391,7 @@ require("../../../html5-common/js/utils/constants.js");
      */
     this.getCurrentTime = function() {
       this.callToFlash("getCurrentTime");
+      return currentTime;
```

Comment:
CURRENT_TIME event is dispatched from AS code to osmf_flash.js  and it gets updated in onCallback = _.bind(function(data)